code/modules/jobs/job_types/cargo_technician.dm

```diff
@@ -9,6 +9,9 @@
 	supervisors = "the quartermaster and the head of personnel"
 	selection_color = "#dcba97"
 	chat_color = "#D3A372"
+	minimal_player_age = 1
+	exp_requirements = 120
+	exp_type = EXP_TYPE_CREW
```

Comment:
2 hours to be a cargo tech?
Cargo tech is one of the new player roles, with nothing to do other than push crates (your fuck ups don't really impact people much).
Cargo techs is one of the first ones I played.

Reply:
Yeah, I'm not really sure what I was thinking here. This is an easy job for even a totally new player.

If/when cargo becomes relevant again, the station will start having expectations if it's workers again I would hope though.